src/screens/OrgList/OrgList.tsx

```
@@ -426,28 +436,24 @@
-                orgsData?.organizationsConnection.map(
-                  (item: InterfaceOrgConnectionInfoType) => {
-                    if (isAdminForCurrentOrg(item)) {
-                      return (
-                        <div key={item._id} className={styles.itemCardOrgList}>
-                          <OrgListCard data={item} />
-                        </div>
-                      );
-                    }
-                  },
-                )}
+              ? orgsData?.edges.map((item: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)